src/lib/Tab/ConditionalTabInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Tab;
+
+/**
+ * Conditional Tab interface needs to be implemented by tabs,
+ * which needs to be present or not, depends on context.
+ */
+interface ConditionalTabInterface
+{
+    /**
+     * Get information about tab presence.
+     *
+     * @param array $parameters
+     *
+     * @return bool
+     */
+    public function getPresence(array $parameters): bool;
```

Comment:
Proposition: `isVisible()`?

Reply:
To hide or display is not responsibility of this interface. `evaluate` would be more proper name here.